app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,9 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: UserVariable[:CONFEMAIL]
+  begin
+    default from: UserVariable[:conference_email]
```

Comment:
Prior to this our app could not be deployed, when running in production Rails eager loads the app code during it's boot, that means during boot it would load this file, but since the UserVariable did not exist it would raise an ArgumentError and therefore not be able to start. In development this file would only be autoloaded when it's referenced, so it would not prevent the app from booting in dev.

Reply:
Good catch, these variables were giving me a headache as well. 👍

Reply:
Took me a while to remember why I didn't use a rescue on these, and its because otherwise you don't get notified about it missing. If variable doesn't exist, It raises error which is easy to read and remember "ohh, I need to add this user variable via admin".
 
For this case its necessary to start the app/run tests so its a great call.

For next conf they will probably just have an internal task to update all user variables, so this shouldn't be a problem in the future.

Reply:
Yeah! That makes sense, I also like the idea of failing early the in case of a missing variable (in this case sadly there was no notification).

I realized now that if admins change the value of this variable they will need to restart the app in order for it to make a difference, that might cause some major confusion, so I'm wondering if it might possibly be better to remove this variable entirely. But what do you think?

Reply:
Hey! Any thoughts on this?

Reply:
Hi, sorry I missed replying to this.

Hum I think you are right, if it requires an app restart, then we can't make it a user variable, since the user doesn't have a way to restart the app. :/ It makes it waaay too confusing.
I didn't think of this as in development it reloads all the time...

I think that instead of the ability to add user variables for stuff like this, we could give the client something like a translation file with all the settings they would probably want to change once a conference.

Reply:
Okay thanks for your input! I think I will go ahead and remove that user variable for now.